supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -21,6 +21,28 @@
 
 *See also*:
 
+[discrete]
+[[network-interface-card]]
+==== image:images/no.png[no] network interface card
+*Description*: A dispreferred referent for the acronym "NIC". Use "network interface controller" instead.
+
+*Use it*: no
+
+*Incorrect forms*:
+
+*See also*: xref:network-interface-controller[network interface controller]
+
+[discrete]
+[[network-interface-controller]]
+==== image:images/yes.png[yes] network interface controller (NIC)
+*Description*: 
```

Comment:
SWAG: The physical or virtual hardware that provides Ethernet connectivity between a host or virtual machine and a network.